content/technical/open-source-as-a-service.md

```diff
@@ -0,0 +1,153 @@
+Title: Your Service is not Open Source
+Date: 2020-02-07 18:35
+Tags: open source, technical
+Slug: open-source-as-a-service
+Category: Technical
+Summary: Sharing the source code to your SaaS is insufficient. We effectively enable users to contribute to the a running service for it to be Open Source.
+
+Sharing the source code to your SaaS is insufficient to make it
+actually be Open Source. Sounds self-contradictory?
+
+Most services that espouse "Open Source", do so by simply
+[throwing the code over the wall](https://github.com/travis-ci/travis-ci).
+It's better than nothing, but really misses the point that powers
+Open Source: enabling users to make a change to the software
+they're using.
+
+Some other Open Source services, such as
+[GitLab.com](https://about.gitlab.com/community/contribute/development/)
+or [ElasticSearch](https://www.elastic.co/about/free-and-open) do include
+the tools used to operate/deploy their service. Pause for applause 👏👏👏
+
+But that's also insufficient to actually enable users to become
+contributors effectively.
+
+Much of the value of a service comes from things other than the code.
+It comes from the infrastructure it runs on, the operational processes,
+the monitoring, the backups, the metrics, the high availability, the
+scalability. It comes from the network effect, with other users, with other
+interconnected services, from legal agreements, and so on.
+
+Any non-toy Service is not reproducible.
+
+Launching a clone of the service in order to effectively contribute
+is a high barrier. Certain kinds of contributions clearly are possible.
+But contributors are essentially asked to "fork" the above non-code
+aspects of the service in order to iterate on their changes against
+their real world use cases.
+
+What's more, without true Open Source principles, we are missing a
+way to have a community develop with a center of gravity around
+the service itself. This leads to companies
+[forking the service](https://aws.amazon.com/blogs/opensource/stepping-up-for-a-truly-open-source-elasticsearch)
+in a way that typical open source projects with a true community
+are resilient to.
+
+I believe that if we enable contributions to services, we reap
```

Comment:
let's add 

"I believe that if we enable contributions to services, *rather than just the software*,"

Reply:
Clearer. Thanks.